hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -452,6 +452,14 @@ void GenericSystem::initialize_storage_vectors(
               "The usage of initial_%s has been deprecated. Please use 'initial_value' instead.",
               interface.name.c_str());
           }
+          else
```

Comment:
This warning would be triggered when there's no inital value provided in either the old way(`initial_<state name>`) or the new way(`initial_value`), but then suggests to the user to use the old way. Wouldnt it be better to suggest the new syntax to the user? 

(New syntax was implemented in #598).

Reply:
Thanks once again @wmmc88. Done